bundles/AdminBundle/Resources/public/js/pimcore/helpers.js

```diff
@@ -3288,3 +3288,29 @@ pimcore.helpers.treeDragDropValidate = function (node, oldParent, newParent) {
 
     return true;
 };
+
+pimcore.helpers.sendRequestWithoutDefaultHeaders = function (
```

Comment:
Hi @blankse because of code quality I think we should communicate with the function which parameters are allowed and not just putting all in one object, where you can pass whatever you want.

Reply:
You can use JSDoc to communicate the possible object keys.

Reply:
Resolving: discussed in the daily and would not consider adding/using JSDoc yet, so will merge in the current state.